Makefile

```diff
@@ -54,3 +54,6 @@ clean: destroy-box remove-tmp
 # helper commands
 update-box:
 	@SSL_CERT_FILE=${SSL_CERT_FILE} CURL_CA_BUNDLE=${CURL_CA_BUNDLE} vagrant box update || (echo '\n\nIf you get an SSL error you might be behind a transparent proxy. \nMore info https://github.com/fredrikhgrelland/vagrant-hashistack/blob/master/README.md#proxy\n\n' && exit 2)
+
+prettify:
+	@terraform fmt -recursive && echo Trying to prettify all .tf files. Do  \'git status\' to see which files got prettified.
```

Comment:
Using `git status` to see what's been prettified can be misleading. I imagine people are working when using this command, in which case `git status` will show much more than what prettify necessarily have touched. Does the format-command possibly give any feedback on changed files that we could use?

Reply:
Yeah, you're right. The format-command does not give any feedback unfortunately. Do you think I should remove the whole echo or should we provide a more descriptive text?

Reply:
Damn. Maybe just remove the git status part?

Reply:
Yeah, let's just do that for now